src/test/kotlin/com/atlassian/performance/tools/hardware/Hardware.kt

```diff
@@ -56,26 +56,26 @@ data class Hardware(
     }
 
     /**
-     * [On-Demand pricing](https://aws.amazon.com/ec2/pricing/on-demand/) as of 2019-05-16.
+     * [On-Demand pricing](https://aws.amazon.com/ec2/pricing/on-demand/) as of 2019-09-05 for the Ohio (eu-east-2).
```

Comment:
Side question, @Luke Heinrich has raised some question about our cost estimation. And I realised we are missing:
1. cost of EBS
2. cost of Load Balancer
Maybe we can add those in the future. But both cost of above are quite small compare EC2.

Reply:
That's a good point.
We should add those in the future. TBH the ELB cost is awkward, because it's not time-based, but throughput-based. OTOH we can assume the throughput over time (as we already do) and recalculate it. Alternatively we could switch to [an EC2 load balancer](https://github.com/atlassian/aws-infrastructure/blob/release-2.17.0/src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/loadbalancer/ApacheEc2LoadBalancerFormula.kt).
Raised https://bulldog.internal.atlassian.com/browse/QUICK-330